spec/models/user_spec.rb

```diff
@@ -63,6 +63,7 @@
     user = FactoryBot.create :user, username: " user "
     expect(user.username).to eq "user"
 
+    school = FactoryBot.create :school, name: "Rice University"
```

Comment:
Could also test that the user gets the location and type from the school here.

Reply:
Fixed